File: python/problem-6.py
T = int(input())


def get_first_digit(n):

    first = n

    while(first >= 10):
        first = first / 10

    return first


for i in range(1, T + 1):
    n = int(input())
    first = get_first_digit(n)
    last = n % 10
    sum = first + last
    print("Sum = " + str(sum))

File: c/problem-6/problem-6.c
#include <stdio.h>
#include <math.h>

int get_first_digit(int number)
{
    int first = number;

    while (first >= 10)
    {
        first = first / 10;
    }

    return first;
}

int main()
{
    int T, first, last, n, sum, i;
    scanf("%d", &T);

    for (i = 1; i <= T; i++)
    {
        scanf("%d", &n);
        first = get_first_digit(n);
        last = n % 10;
        sum = first + last;

        printf("Sum = %d\n", sum);
    }
}